### PR TITLE
Add Kindakeyboards Conone65 Via support

### LINKED ISSUE #22319: [Feature Request] Add VIA support for Kindakeyboards Conone65
### Feature Request Type

- [ ] Core functionality
- [ ] Add-on hardware support (eg. audio, RGB, OLED screen, etc.)
- [ ] Alteration (enhancement/optimization) of existing feature(s)
- [X] New behavior

### Description

This is a request to add Via support for the Conone65 from Kinda Keyboards.

I've actually started working on this already (including the separate Via changes) with the blessing of the keyboard designer. I will tag the maintainer when I have the pull request ready and available.

### PR DESCRIPTION
**Pull request by mattchan**
## Description

This change addresses #22319 by adding a new Via keymap and `rules.mk` files for the Conone65. Additionally, there is minor syntax cleanup in the default keymaps for this board, and a `README` link for the hardware availability has been updated since the order form for the board is no longer open.

The Via keymap is nearly the same as the default keymap though with some additional spacing (mainly for my own visual purposes when writing the Via JSON file)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Fixes #22319

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mattchan** [comment]
@evyd13, I've added a keymap for the Via layout for your awareness.

File: keyboards/kindakeyboards/conone65/keymaps/via/readme.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Conone65
+
```
Comment (mattchan) [inline keyboards/kindakeyboards/conone65/keymaps/via/readme.md:2]: I didn't add an Imgur link to the keyboard which was in the template, but I have asked the keyboard designer if I can reuse one of the images from the Kindakeyboards website. Is the image absolutely required before merging?

File: keyboards/kindakeyboards/conone65/keymaps/via/readme.md
Comment (zvecr) [inline keyboards/kindakeyboards/conone65/keymaps/via/readme.md:1]: This file should be removed.

File: keyboards/kindakeyboards/conone65/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_BASE] = LAYOUT_all(
+        QK_GESC, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,   KC_6,     KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_BSPC, KC_HOME,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,   KC_Y,     KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,          KC_BSLS, KC_PGUP,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,   KC_H,     KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_BSLS,          KC_ENT,  KC_PGDN,
+        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,   KC_B,     KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,          KC_UP,   KC_END,
+        KC_LCTL, KC_LGUI, KC_LALT,          KC_SPC,          KC_SPC,            KC_SPC,  KC_RALT, KC_APP,  KC_RCTL, KC_LEFT,          KC_DOWN, KC_RGHT
+    ),
+    [_FN] = LAYOUT_all(
```
Comment (zvecr) [inline keyboards/kindakeyboards/conone65/keymaps/via/keymap.c:32]: This layer is not accessible so should either be removed or something like `MO(1)` added to the base layer.

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/kindakeyboards/conone65/keymaps/via/readme.md
Comment (mattchan) [reply keyboards/kindakeyboards/conone65/keymaps/via/readme.md:1]: Good catch. I had noticed that but forgot about it.

File: keyboards/kindakeyboards/conone65/keymaps/via/keymap.c (hunk 2 above)
Comment (mattchan) [reply keyboards/kindakeyboards/conone65/keymaps/via/keymap.c:32]: Ah, thank you. I will change that.

Comment (mattchan) [reply keyboards/kindakeyboards/conone65/keymaps/via/keymap.c:32]: Updated to include `MO(1)` on the base layer.

File: keyboards/kindakeyboards/conone65/keymaps/via/readme.md
Comment (mattchan) [reply keyboards/kindakeyboards/conone65/keymaps/via/readme.md:1]: File has been removed.

Comment (zvecr) [reply keyboards/kindakeyboards/conone65/keymaps/via/readme.md:1]: You have not removed this file and removed something else.

Comment (mattchan) [reply keyboards/kindakeyboards/conone65/keymaps/via/readme.md:1]: I misread. I'll remove this and update the main one.

File: keyboards/kindakeyboards/conone65/keymaps/via/readme.md (hunk 1 above)
Comment (waffle87) [reply keyboards/kindakeyboards/conone65/keymaps/via/readme.md]: Not required, but helpful to have, specifically of the PCB. Be sure to host it on Imgur if you are able to add an image.

Comment (mattchan) [reply keyboards/kindakeyboards/conone65/keymaps/via/readme.md]: Thanks, I'll add an image and update in the next few days.

Comment (mattchan) [reply keyboards/kindakeyboards/conone65/keymaps/via/readme.md]: I've added an Imgur link for the PCB.

**Comment by mattchan** [comment]
Should I rebase and squash my commits manually, or will this be taken care of by the GitHub merge workflow after approval?

**Review by waffle87 — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**